addons/udes_stock/models/stock_move.py

```diff
@@ -1,10 +1,31 @@
 # -*- coding: utf-8 -*-
-from odoo import models, api, _
+from odoo import models, fields, api, _
+from odoo.addons import decimal_precision as dp
 
 
 class StockMove(models.Model):
     _inherit = "stock.move"
 
+    u_initial_demand = fields.Float("Initial Demand", digits="Product Unit of Measure")
```

Comment:
Is this the move UoM? Can we just make it clearer - looks to me like the move UoM because we use product_uom_qty

Reply:
Yeah let's rename it to u_uom_initial_demand